### DISCUSSION TITLE
Feature to add individual users to github teams using github api

### DISCUSSION THREAD

**Discussion by githubofkrishnadhas** [community/community#83904] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Hello team,

I have been learning on github actions , workflows, apis practical usages. And i came to see a feature that can improve the current rest API. This is for API endpoints for github teams.

currently we dont have a option to add members to a github teams using rest apis. atleast i was unable to see it even though i asked one of my friends who is good at this stuff. at the same time if thats needed we can add the users as maintainers. but that is not a good practice on Cloud nor anywhere. we will always promote least access based principle.

if there is a way we can add individual user to a github teams not as its maintainers / owners please share the documentation for my knowledge and it could be really helpful.

if this is not there, please consider this as a humble feature request which will reduce a lot of work in future who are depending on automations especially using github apis.

Thanks,
Krishnadhas
@githubofkrishnadhas

**Comment by githubofkrishnadhas** [comment]
I have been looking at these documentations:
* https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28
*https://docs.github.com/en/rest/teams/teams?apiVersion=2022-11-28

**Comment by Murodxojayev1ch** [comment]
Response #1
